lib/apollo/hooks/state/currentUser.js

```diff
@@ -1,14 +0,0 @@
-import { useQuery } from '@apollo/client';
-import CurrentUser from 'graphql/queries/currentUser.graphql';
-
-export const useCurrentUser = (prefetch = true) => {
```

Comment:
We use useCurrentUser
<img width="542" alt="Screenshot 2021-11-30 at 15 04 14" src="https://user-images.githubusercontent.com/14818162/144044142-8a792177-d4e3-493d-bbaa-cb17938f2083.png">

Reply:
yes, we had 2 files: `currentUser.ts` and `currentUser.js`. I've left `.ts` file